ngraph/core/src/op/loop.cpp

```diff
@@ -15,6 +15,7 @@
 //*****************************************************************************
 
 #include "ngraph/op/loop.hpp"
+#include <validation_util.hpp>
```

Comment:
is this include needed?

Reply:
I use `ngraph::normalize_axis` from this header